packages/core/src/components/group-item/group-item.scss

```diff
@@ -32,6 +33,7 @@ $focusBorderWidth: 1px;
     background-color: var(--theme-group-item--background);
     border: 1px solid transparent;
     color: var(--theme-color-std-text);
+    cursor: inherit;
```

Comment:
Only show _pointer_ if _suppressSelection_ is not set!

Reply:
yes I will do it

Reply:
Great, can you do it today? :D
We are planning to release this week.

Reply:
of course I will do it today :)